src/control_node_startup_script.sh

```diff
@@ -102,6 +90,8 @@ helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
 helm repo update
 export RELEASE=jhub
 export NAMESPACE=jhub
+JUPYTERHUB_IMAGE="jupyterhub/jupyterhub"
+kubectl create namespace $NAMESPACE
```

Comment:
I guess this step is now needed because of the upgrade to helm 3, right?
If not, why you add it?

Reply:
This was a warning. Not strictly necessary,  but doing it "the preferred way".

Reply:
Yep, the helm command should be creating the namespace, but I like the idea to clean up the warning (maybe you can add a brief comment about this?)